lib/ansible/modules/cloud/amazon/rds_param_group.py

```diff
@@ -165,31 +181,47 @@
     'aurora5.6',
     'mariadb10.0',
     'mariadb10.1',
+    'mariadb10.2',
```

Comment:
It also doesn't look like `VALID_ENGINES` is ever used, so we could remove that. Or if we want to use it, not hardcoding the list would be better. It also looks like with this update the list is still missing a bunch of stuff.
```
>>> import boto3
>>> c = boto3.client('rds', 'us-east-1')
>>> all_allowed_db_engines = [stuff['DBParameterGroupFamily'] for stuff in c.get_paginator('describe_db_engine_versions').paginate().build_full_result()['DBEngineVersions']]
>>> print(len(all_allowed_db_engines))
242
>>> print(all_allowed_db_engines)
['aurora-mysql5.7', 'neptune1', 'aurora-postgresql9.6', 'aurora-postgresql9.6', 'aurora-postgresql9.6', 'aurora-postgresql9.6', 'aurora-postgresql10', 'mariadb10.0', 'mariadb10.0', 'mariadb10.0', 'mariadb10.0', 'mariadb10.0', 'mariadb10.0', 'mariadb10.0', 'mariadb10.1', 'mariadb10.1', 'mariadb10.1', 'mariadb10.1', 'mariadb10.1', 'mariadb10.1', 'mariadb10.2', 'mariadb10.2', 'mariadb10.2', 'mariadb10.3', 'mysql5.5', 'mysql5.5', 'mysql5.5', 'mysql5.5', 'mysql5.5', 'mysql5.5', 'mysql5.6', 'mysql5.6', 'mysql5.6', 'mysql5.6', 'mysql5.6', 'mysql5.6', 'mysql5.7', 'mysql5.7', 'mysql5.7', 'mysql5.7', 'mysql5.7', 'mysql5.7', 'mysql8.0', 'oracle-ee-11.2', 'oracle-ee-11.2', 'oracle-ee-11.2', 'oracle-ee-11.2', 'oracle-ee-11.2', 'oracle-ee-11.2', 'oracle-ee-11.2', 'oracle-ee-11.2', 'oracle-ee-11.2', 'oracle-ee-11.2', 'oracle-ee-11.2', 'oracle-ee-11.2', 'oracle-ee-11.2', 'oracle-ee-11.2', 'oracle-ee-11.2', 'oracle-ee-11.2', 'oracle-ee-11.2', 'oracle-ee-12.1', 'oracle-ee-12.1', 'oracle-ee-12.1', 'oracle-ee-12.1', 'oracle-ee-12.1', 'oracle-ee-12.1', 'oracle-ee-12.1', 'oracle-ee-12.1', 'oracle-ee-12.1', 'oracle-ee-12.1', 'oracle-ee-12.1', 'oracle-ee-12.1', 'oracle-ee-12.1', 'oracle-ee-12.1', 'oracle-ee-12.2', 'oracle-se-11.2', 'oracle-se-11.2', 'oracle-se-11.2', 'oracle-se-11.2', 'oracle-se-11.2', 'oracle-se-11.2', 'oracle-se-11.2', 'oracle-se-11.2', 'oracle-se-11.2', 'oracle-se-11.2', 'oracle-se-11.2', 'oracle-se-11.2', 'oracle-se-11.2', 'oracle-se-11.2', 'oracle-se-11.2', 'oracle-se-11.2', 'oracle-se-11.2', 'oracle-se1-11.2', 'oracle-se1-11.2', 'oracle-se1-11.2', 'oracle-se1-11.2', 'oracle-se1-11.2', 'oracle-se1-11.2', 'oracle-se1-11.2', 'oracle-se1-11.2', 'oracle-se1-11.2', 'oracle-se1-11.2', 'oracle-se1-11.2', 'oracle-se1-11.2', 'oracle-se1-11.2', 'oracle-se1-11.2', 'oracle-se1-11.2', 'oracle-se1-11.2', 'oracle-se1-11.2', 'oracle-se2-12.1', 'oracle-se2-12.1', 'oracle-se2-12.1', 'oracle-se2-12.1', 'oracle-se2-12.1', 'oracle-se2-12.1', 'oracle-se2-12.1', 'oracle-se2-12.1', 'oracle-se2-12.1', 'oracle-se2-12.1', 'oracle-se2-12.1', 'oracle-se2-12.1', 'oracle-se2-12.1', 'oracle-se2-12.2', 'aurora5.6', 'postgres9.3', 'postgres9.3', 'postgres9.3', 'postgres9.3', 'postgres9.3', 'postgres9.3', 'postgres9.3', 'postgres9.3', 'postgres9.3', 'postgres9.4', 'postgres9.4', 'postgres9.4', 'postgres9.4', 'postgres9.4', 'postgres9.4', 'postgres9.4', 'postgres9.4', 'postgres9.4', 'postgres9.5', 'postgres9.5', 'postgres9.5', 'postgres9.5', 'postgres9.5', 'postgres9.5', 'postgres9.5', 'postgres9.5', 'postgres9.5', 'postgres9.6', 'postgres9.6', 'postgres9.6', 'postgres9.6', 'postgres9.6', 'postgres9.6', 'postgres9.6', 'postgres9.6', 'postgres10', 'postgres10', 'postgres10', 'postgres10', 'sqlserver-ee-10.5', 'sqlserver-ee-10.5', 'sqlserver-ee-10.5', 'sqlserver-ee-11.0', 'sqlserver-ee-11.0', 'sqlserver-ee-11.0', 'sqlserver-ee-11.0', 'sqlserver-ee-12.0', 'sqlserver-ee-12.0', 'sqlserver-ee-12.0', 'sqlserver-ee-13.0', 'sqlserver-ee-13.0', 'sqlserver-ee-13.0', 'sqlserver-ee-13.0', 'sqlserver-ee-13.0', 'sqlserver-ee-13.0', 'sqlserver-ee-14.0', 'sqlserver-ee-14.0', 'sqlserver-ee-14.0', 'sqlserver-ex-10.5', 'sqlserver-ex-10.5', 'sqlserver-ex-10.5', 'sqlserver-ex-11.0', 'sqlserver-ex-11.0', 'sqlserver-ex-11.0', 'sqlserver-ex-11.0', 'sqlserver-ex-12.0', 'sqlserver-ex-12.0', 'sqlserver-ex-12.0', 'sqlserver-ex-12.0', 'sqlserver-ex-13.0', 'sqlserver-ex-13.0', 'sqlserver-ex-13.0', 'sqlserver-ex-13.0', 'sqlserver-ex-13.0', 'sqlserver-ex-13.0', 'sqlserver-ex-14.0', 'sqlserver-ex-14.0', 'sqlserver-ex-14.0', 'sqlserver-se-10.5', 'sqlserver-se-10.5', 'sqlserver-se-10.5', 'sqlserver-se-11.0', 'sqlserver-se-11.0', 'sqlserver-se-11.0', 'sqlserver-se-11.0', 'sqlserver-se-12.0', 'sqlserver-se-12.0', 'sqlserver-se-12.0', 'sqlserver-se-12.0', 'sqlserver-se-13.0', 'sqlserver-se-13.0', 'sqlserver-se-13.0', 'sqlserver-se-13.0', 'sqlserver-se-13.0', 'sqlserver-se-13.0', 'sqlserver-se-14.0', 'sqlserver-se-14.0', 'sqlserver-se-14.0', 'sqlserver-web-10.5', 'sqlserver-web-10.5', 'sqlserver-web-10.5', 'sqlserver-web-11.0', 'sqlserver-web-11.0', 'sqlserver-web-11.0', 'sqlserver-web-11.0', 'sqlserver-web-12.0', 'sqlserver-web-12.0', 'sqlserver-web-12.0', 'sqlserver-web-12.0', 'sqlserver-web-13.0', 'sqlserver-web-13.0', 'sqlserver-web-13.0', 'sqlserver-web-13.0', 'sqlserver-web-13.0', 'sqlserver-web-13.0', 'sqlserver-web-14.0', 'sqlserver-web-14.0', 'sqlserver-web-14.0']
```

Reply:
Oh, that looks like that contains a bunch of duplicates, so maybe this *is* everything.
```
>>> print(len(set(all_allowed_db_engines)))
45
>>> print(set(all_allowed_db_engines))
set(['sqlserver-web-14.0', 'sqlserver-web-11.0', 'sqlserver-ee-10.5', 'sqlserver-web-10.5', 'oracle-se2-12.1', 'oracle-se2-12.2', 'sqlserver-web-13.0', 'sqlserver-web-12.0', 'sqlserver-ee-13.0', 'oracle-ee-12.2', 'oracle-ee-12.1', 'sqlserver-ee-12.0', 'sqlserver-ex-12.0', 'aurora-postgresql10', 'aurora5.6', 'sqlserver-ex-13.0', 'oracle-ee-11.2', 'oracle-se1-11.2', 'sqlserver-se-14.0', 'sqlserver-ee-11.0', 'sqlserver-ex-10.5', 'aurora-mysql5.7', 'neptune1', 'mysql8.0', 'sqlserver-ex-11.0', 'oracle-se-11.2', 'postgres9.6', 'mysql5.5', 'mysql5.7', 'mysql5.6', 'postgres9.5', 'sqlserver-ex-14.0', 'postgres9.4', 'postgres9.3', 'mariadb10.2', 'mariadb10.3', 'mariadb10.0', 'mariadb10.1', 'aurora-postgresql9.6', 'sqlserver-ee-14.0', 'sqlserver-se-13.0', 'sqlserver-se-11.0', 'sqlserver-se-12.0', 'sqlserver-se-10.5', 'postgres10'])
```

Reply:
It looks like these are missing:
['neptune1', 'aurora-postgresql9.6', 'aurora-mysql5.7', 'aurora-postgresql10']
And these should not be in the list:
['oracle-se-12.1', 'oracle-se1-12.1', 'mysql5.1']

Reply:
I think it would be okay to point to the boto3 docs or CLI, but if you just want to update those LGTM.